### PR TITLE
[quest] Add 'Forces of Nature: Mountain Giants' Objectives

### LINKED ISSUE #6241: Quest Issue for quest 25469 - Forces of Nature: Mountain Giants (Feralas, Cataclysm Classic)
## Description
Mob relevant for the quest is not marked on the map or in the mob tooltip

## Wowhead link
Quest: https://www.wowhead.com/cata/quest=25469/forces-of-nature-mountain-giants
Mob: https://www.wowhead.com/cata/npc=40026/mountain-giant

## Screenshots
NPC not marked as relevant for quest:
image

## Questie & WoW version
Questie v10.3.7 & Cataclysm Classic

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #6241

## Proposed changes

- Add a fix to include correct `objectives` for 'Forces of Nature: Mountain Giants'

### CODE REVIEW

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -2962,6 +2962,9 @@ function CataQuestFixes.Load()
         [25467] = { -- Kliklak's Craw
             [questKeys.startedBy] = {{40276},nil,{54345}},
         },
+        [25469] = { -- Forces of Nature: Mountain Giants
+            [questKeys.objectives] = {nil,nil,nil,nil,{{{40026},40026,"Mountain Giants Rallied",Questie.ICON_TYPE_INTERACT}}},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:2966]: A `killCreditObjective` with only a single ID does not make sense, over using a simple "monster" objective

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/cataQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/cataQuestFixes.lua]: https://github.com/Questie/Questie/pull/6255/commits/18fcf433739b13bd5027cc3de716a350d90938e8

File: Database/Corrections/cataQuestFixes.lua [hunk 2]
```diff
@@ -3005,6 +3005,9 @@ function CataQuestFixes.Load()
         [25468] = { -- Forces of Nature: Faerie Dragons
             [questKeys.objectives] = {nil,nil,nil,nil,{{{5276,5278},5276,"Faerie Dragons Rallied",Questie.ICON_TYPE_INTERACT}}},
         },
+        [25469] = { -- Forces of Nature: Mountain Giants
+            [questKeys.extraObjectives] = {{nil,Questie.ICON_TYPE_MOUNT_UP,l10n("Mountain Giants Rallied"),0,{{"monster",40026}}}},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:3009]: `objective` was correct already. But not of type "killCredit" (5th index) but of type "monster" (1st index):

```suggestion
            [questKeys.objectives] = {{{40026}}},
```

Comment (Sommos) [reply Database/Corrections/cataQuestFixes.lua]: https://github.com/Questie/Questie/pull/6255/commits/eaccca163f9fc1cd1a0ed728367bd2abb5991552

**Pull request merged**